cmd/woodpecker-autoscaler/flags.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"os"
 
-	"github.com/urfave/cli/v2"
+	"github.com/urfave/cli/v3"
```

Comment:
v3 has some breaking changes to the argument order, etc. Would this require us to mark this PR as breaking as well?

Reply:
For example? Im not aware of any impact to end users.

Reply:
Thanks I see. Im also fine to just bump major.